drv/lpc55-update-server/src/main.rs

```diff
@@ -462,7 +494,7 @@ impl ServerImpl<'_> {
         &mut self,
     ) -> Result<(SlotId, Option<SlotId>, Option<SlotId>), UpdateError> {
         let (cfpa_word_number, cfpa_version) =
-            self.cfpa_word_number_and_version()?;
+            self.cfpa_word_number_and_version(false)?;
```

Comment:
Nit: I dislike passing bools, because you don't know what it means at the call site. However, clearly it can make the code more succinct inside the function than an enum. To work across that tension, many of us on the control plane side declare a temporary named boolean variable and then pass that into the method rather than the literal `true` or `false`.